scripts/app.js

```diff
@@ -89,8 +93,55 @@ document.addEventListener("keydown", function(event) {
 
         saveContentToCache();
     }
+
+    if (event.ctrlKey && event.code === "Slash") {
+        event.preventDefault();
+        zoom(5);
+    }else if (event.ctrlKey && event.code === "Equal") {
+        event.preventDefault();
+        zoom(-5);
+    }else if (event.ctrlKey && event.code === "Digit0") {
+        event.preventDefault();
+        resetZoom();
+    }
+});
+
+// Zoom text
+buttonZoomPlus.addEventListener("click", function(event) {
+    event.preventDefault();
+    zoom(5);
+});
+
+buttonZoomMinus.addEventListener("click", function(event) {
+    event.preventDefault();
+    zoom(-5);
 });
 
+buttonZoomReset.addEventListener("click", function(event) {
+    event.preventDefault();
+    resetZoom();
+});
+
+function zoom(zoomValue) {
+    const currentFontSize = parseInt(window.getComputedStyle(input).fontSize);
+
```

Comment:
`currentFontSize + zoomValue` is repeated 4 times in the function: you can make it a constant.

```
const newFontSize = currentFontSize + zoomValue
```